src/Microsoft.Azure.WebJobs.Extensions.Kafka/KafkaEventDataHeaders.cs

```diff
@@ -0,0 +1,101 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Microsoft.Azure.WebJobs.Extensions.Kafka
+{
+    public class KafkaEventDataHeaders : IKafkaEventDataHeaders
+    {
+        List<IKafkaEventDataHeader> headers = new List<IKafkaEventDataHeader>();
+        internal readonly bool IsReadOnly;
```

Comment:
Member variables should start with lower case. If we want to make a property we should add a getter.

Reply:
Done